### PR TITLE
Milestone3: Bootstrap columns and borders

### PR DESCRIPTION
**Pull request by Abhi-1725**
We added the following to milestone 2:

- Create these sections of the website for the 3 screen sizes
- Completed the remaining sections after milestone2

### CODE REVIEW

File: newsweek.css [hunk 1]
```diff
@@ -117,19 +117,19 @@ main {
...
-}
+} */
 
-.btn {
+/* .btn {
   height: 2rem;
   color: black;
   font-weight: bolder;
-}
+} */
```
Comment (SajjadAhmad14) [inline newsweek.css:132]: Remove inline comments

File: newsweek.css [hunk 2]
```diff
@@ -141,19 +141,19 @@ main {
   opacity: 0.75;
 }
 
-.footer-color {
+/* .footer-color {
   color: var(--gray);
   text-decoration: none;
-}
+} */
```
Comment (SajjadAhmad14) [inline newsweek.css:147]: Remove inline comments.

File: newsweek.css [hunk 3]
```diff
@@ -141,19 +141,19 @@ main {
...
 .article-title {
   color: rgba(3, 3, 3, 0.4);
   font-size: 1.1rem;
 }
 
-.fs-7 {
+/* .fs-7 {
   font-size: 0.9rem;
-}
+} */
```
Comment (SajjadAhmad14) [inline newsweek.css:156]: Remove inline comments.

File: newsweek.css (hunk 1 above)
Comment (Abhi-1725) [reply newsweek.css]: I have removed the inline comments from the CSS file.

File: newsweek.css (hunk 2 above)
Comment (Abhi-1725) [reply newsweek.css]: I have removed the inline comments from the CSS file.

File: newsweek.css (hunk 3 above)
Comment (Abhi-1725) [reply newsweek.css]: I have removed the inline comments from the CSS file.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## But there are some changes to make in order to get your project approved.

- [x]  README does not include the instructions to run the project locally, also the live link is broken. Please fix that.
- [x]  Please include files in gitignore which are not used e.g vscode.
- [x] Please remove commented code from CSS file.  
- [ ] You should group multiple CSS classes if they have the same rules. Always try to follow the DRY rule.
- [ ] Each title in the magazine and Editor's pick section uses a hx tag and has a link. The title on the hover shows text decoration.
- [x] Navbar is broken on Laptop 1024px screen.
broken-nav

- [x] Orignal page has two columns in each row on tablet viewport in Editor's pick section, but your page has four columns. Kindly fix it!
- [ ] SUBSCRIBE button in the navbar is broken on small mobile screen size. Moreover, the login icon has two more small icons attached. Please fix it!
broken-icon
- [ ] Sponsored Insight section is missing an image with a link on each story on the medium and small screen sizes.
### Main newspaper sections (US, World, etc.):
- [x] Each image have a link.
- [x] Each title uses a hx tag and have a link.
### Subscribe / Newsletter 
- [x] Subscribe section should have a grey background.
- [x] Each section title uses a hx tag 
- [x] Please make Subscribe and Newsletter sections exactly the same as the original on medium and small screen sizes.

# Optional
- [ ] You can use a mock like [this](https://techsini.com/multi-mockup/index.php) to add screenshots of your project on different screen sizes. 

### Happy Coding!

## You can always connect with me on slack ( @Sajjad Ahmad ).

File: index.html [hunk 4]
```diff
@@ -159,1853 +144,2154 @@
...
-        </section>
-      </nav>
+        <div class="col-md-3 col-lg-3">
+          <div class="img1">
+            <a href="#"><img src="img/cover-news.JPG" class="img-2 img-fluid" width="300" height="230" alt="cover-news"></a>
+          </div>
+          <div class="cover">
+            <a href="#" class="bg-danger text-white text-decoration-none"><b class="text-white">COVER</b></a> <a href="#" class="text-danger text-decoration-none"><b class="text-danger">NEWS</b></a>
+            <p class="border border-light">
+              <a href="#" class="text-dark fw-bold fs-6 red-underline text-decoration-none">On His Way Out, Trump Trashes America—and the GOP</a>
```
Comment (Berabjesus) [inline index.html:1584]: - [ ]  This  anchor tag should be a hx tag and has a link. Something like ` <h1><a href="#">link</a></h1>`

File: index.html [hunk 5]
```diff
@@ -159,1853 +144,2154 @@
...
+        <a href="#" class="sub-title-1 fs-4 fw-bold text-danger text-decoration-none ">U.S.</a>
+      </div>
+      <br>
+      <div class="container-fluid row justify-content-between">
+        <div class="col-12 col-md-3 col-lg-3 border border-white">
+          <div class="img1">
+            <img src="img/homeless.JPG" class="img-17 img-fluid" alt="homeless" width="400" height="300">
+          </div>
+          <div class="">
+            <a href="#" class="card-text text-dark fw-bold fs-5 text-decoration-none">Indiana Homeless Man Chooses to Be Arrested to Avoid Cold Weather</a>
```
Comment (Berabjesus) [inline index.html:1788]: - [ ]  The text content should use a hx tag and has a link. Something like ` <h1><a href="#">link</a></h1>`

File: index.html [hunk 6]
```diff
@@ -159,1853 +144,2154 @@
...
+      <hr>
+
+      <div class=" container-fluid px-4">
+        <a href="#" class="sub-title-1 fs-4 fw-bold text-danger text-decoration-none ">U.S.</a>
+      </div>
+      <br>
+      <div class="container-fluid row justify-content-between">
+        <div class="col-12 col-md-3 col-lg-3 border border-white">
+          <div class="img1">
+            <img src="img/homeless.JPG" class="img-17 img-fluid" alt="homeless" width="400" height="300">
```
Comment (Berabjesus) [inline index.html:1785]: - [ ]  This image tag should have a link.

**Review by Berabjesus — CHANGES_REQUESTED** [review]
## Status: CHANGES REQUIRED :warning: :construction:
Hey, @Abhi-1725  and @P-Rutkowski you have done an awesome job so far. Great to see that you have worked on some of the requested features by the previous TSE.

### However, based on the previous review, you still do have a few changes to carry out before you proceed so please take a look at the things you have to work on below.

## readme related
- [ ] README still does not include the instructions to run the project locally. Include some terminal commands (like `git clone --repo--`) so a non-technical user can read, understand and clone your project locally. 
- [ ] The live demo link does not work.
- [ ] .vscode should not appear in the project if it's properly added to `.gitignore` file. Use `.vscode/`

## HTML
- [ ] Each title in the magazine and Editor's pick section uses a hx tag and has a link. The title on the hover shows text decoration. (see comment below)
- [ ] Navbar is still broken on Laptop 1024px screen. 
image

- [ ] Editors pick section should have two columns in tablet mode. Yours is not responsive.
#### original 👇 
image
#### yours 👇 
image

### Main newspaper sections (US, World, etc.):
- [ ] Each image should have a link.
- [ ] Each title should use a hx tag and should have a link.

### Subscribe / Newsletter
- [ ] Subscribe section should have a grey background.
- [ ]  Each section title uses a hx tag
- [ ] Please make Subscribe and Newsletter sections exactly the same as the original on medium and small screen sizes. It looks a bit messy at the moment.

## optional
- [ ] The navbar should be fixed to the top.

### **❗ You have one (1) remaining review. As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), when you reach to the limit of 3 code reviews and your project was not approved, you will need to repeat the project. If you think that the code review was not fair, you can [request a second-opinion using this form.](https://airtable.com/shrmBIv6cXYrq4Fgv)**

#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

File: index.html (hunk 5 above)
Comment (Abhi-1725) [reply index.html:1733]: I have used bootstrap classes instead of headline tag,  as this is bootstrap oriented.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Abhi-1725 @P-Rutkowski, good job on your milestone #3! 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

You can contact me on slack if you have any questions, just search for me as @ Miguel Ricaño.

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

**Pull request merged**